src/styles/index.css

```
@@ -0,0 +1,98 @@
+@import "./reset.css";
+@import "./stars.css";
+
+:root {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  box-sizing: border-box;
+  width: 200px;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin: 0 0 4px;
```

Can you add just margin-bottom?
```suggestion
  margin-bottom: 4px;
```